### PR TITLE
[PostRector] Fixing auto import to not placed before top docblock

### PR DESCRIPTION
**Pull request by samsonasik**
Fixes https://github.com/rectorphp/rector/issues/6833

### CODE REVIEW

**Comment by samsonasik** [comment]
Fixed 🎉

File: tests/Issues/AutoImportBeforeDocblock/Fixture/fixture.php.inc [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+<?php
+
+namespace Rector\Core\Tests\Issues\AutoImportBeforeDocblock\Fixture;
+
+/**
+ * License
+ */
+use Foobar\B;
+
+use Foobar\A;
```
Comment (samsonasik) [inline tests/Issues/AutoImportBeforeDocblock/Fixture/fixture.php.inc]: as docblock is belong to the use statement, this seems the possible fix for it.

**Comment by samsonasik** [comment]
All checks have passed 🎉 @TomasVotruba it is ready for review.

**Comment by TomasVotruba** [comment]
Thank you :+1:

**Pull request merged**